joel/lab1/part-1/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.15)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
```

Comment:
prefer to use setting a specific target's compile options: https://cmake.org/cmake/help/latest/command/target_compile_options.html